clutch_mega_driver:
  ros__parameters:
    serial_port: '/dev/ttyUSB0'
    baud_rate: 115200
    polling_rate: 100.0
    motor_index_remapping: [4,1,7,2,0,3,5,6]
